docs/changehistory/NextVersion.md

```diff
@@ -3,3 +3,14 @@ publish: false
 ---
 # NextVersion
 
+## Display
+
+### BENTLEY_materials_point_style
+
+This PR adds support for the proposed [BENTLEY_materials_point_style](https://github.com/CesiumGS/glTF/pull/91) glTF extension.
+
+This PR allows iTwin.js to process and apply the above extension when loading glTF files. This means point primitives will be able to have a diameter property specified and respected in iTwin.js when loaded via glTF.
```

Comment:
"this PR" doesn't make sense in a change summary for an entire minor version of all packages.